.github/workflows/release.yml

```diff
@@ -82,6 +98,30 @@ jobs:
           github_token: ${{ secrets.OPENSOURCE_BOT_TOKEN }}
           branch: develop
 
+      - name: Re-enable branch protection
+        id: enable-branch-protection
+        if: always()
```

Comment:
do we need this line if its `always()`?

Reply:
The `if : always()` tells it to run even if the previous job fails, so no matter what happens it should always re-enable branch protection